ephios/core/views/accounts.py

```diff
@@ -32,6 +34,20 @@ class UserProfileListView(CustomPermissionRequiredMixin, ListView):
     permission_required = "core.view_userprofile"
     ordering = "last_name"
 
+    def get_queryset(self):
+        global_preferences = global_preferences_registry.manager()
+        categories = global_preferences["general__relevant_qualification_categories"]
+        qs = UserProfile.objects.all()
+        for category in categories:
+            qs = qs.prefetch_related(
+                Prefetch(
+                    "qualification_grants",
+                    queryset=QualificationGrant.objects.filter(qualification__category=category),
+                    to_attr=str(category.pk),
```

Comment:
Can we do something more verbose than integer ids? Like `qualification_category_{id}`?

Reply:
it's just a bit easier to access in the template because stuffing strings together there is always such a pain